app/models/repository_owner/gitlab.rb

```diff
@@ -163,6 +163,7 @@ def self.create_org(org_hash)
     def get_json(url)
       r = Typhoeus::Request.new(url,
                                 method: :get,
+                                followlocation: true,                                
```

Comment:
Typhoeus doesn't let you set a limit, but assuming this is safe because the url is always GitLab

Reply:
WDYT about using `PackageManager::ApiService.request_and_parse_json` here?